packages/odf/components/attach-storage-storagesystem/storageclass-form.tsx

```diff
@@ -51,6 +68,55 @@ const StorageClassForm: React.FC<StorageClassFormProps> = ({
     });
   };
 
+  const [data, loaded, loadError] = useK8sWatchResource<
+    StorageClassResourceKind[]
+  >({
+    isList: true,
+    groupVersionKind: getGroupVersionKindForModel(StorageClassModel),
+  });
```

Comment:
these 3 are the current cache key used in `LSOStorageClassDropdown`... so need to keep it same here as well...
```suggestion
  >({
  kind: StorageClassModel.kind,
  namespaced: false,
  isList: true,
  });
```